packages/dashboard/src/components/SetupBanner/SetupCard.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { keyframes } from 'office-ui-fabric-react/lib/Styling';
+
+import { ISetupCardStyles, ISetupCardStylesProps } from './SetupCard.types';
+
+const freTransformIntro = keyframes({
+  from: {
+    animationTimingFunction: 'cubic-bezier(.29,1.35,.71,1.01)',
+    transform: 'translateY(496px)'
+  }
+});
+
+const freOpacityIntro = keyframes({
+  from: {
+    animationTimingFunction: 'cubic-bezier(0.28, 0, 0, 1)'
+  },
+  to: {
+    opacity: '1'
+  }
+});
+
+export const getStyles = (props: ISetupCardStylesProps): ISetupCardStyles => {
+  return {
+    root: {
+      fill: 'none',
+      opacity: '0',
+      animation: `${freTransformIntro} .583s 1, ${freOpacityIntro} .583s 1`,
+      animationFillMode: 'forwards',
+      position: 'absolute'
+    },
+    title: {
+      fill: 'black',
```

Comment:
>black [](start = 13, length = 5)

black, white colors can be pulled from palette and for other colors please see if they are available in palette

Reply:
unfortunately the other colors aren't. We could consider adding our own admin palette though. I will follow up with Sung on this.